pkg/didcomm/protocol/outofband/service.go

```diff
@@ -160,11 +170,37 @@ func (s *Service) HandleInbound(msg service.DIDCommMsg, myDID, theirDID string)
 			},
 			Properties: nil,
 		}
+
+		events <- event
+
+		logger.Debugf("dispatched event: %+v", event)
 	}()
 
 	return "", nil
 }
 
+func sendMsgEvent(t service.StateMsgType, listeners *service.Message, msg service.DIDCommMsg, p *eventProps) {
+	var stateName string
+
+	if msg.Type() == RequestMsgType {
+		stateName = StateRequested
+	}
+
```

Comment:
I am a bit confused about that piece of code. It seems like `StateID` can be empty. Is that expected behavior?

Reply:
@soluchok this should be refactored after the oob invitation msg type is added to the service